aci/file.go

```diff
@@ -123,7 +124,7 @@ func ManifestFromImage(rs io.ReadSeeker) (*schema.ImageManifest, error) {
 		if err != nil {
 			return nil, err
 		}
-		if hdr.Name == "manifest" {
+		if path.Clean(hdr.Name) == "manifest" {
```

Comment:
filepath.Clean?
also, could you please actually change "manifest" to ManifestFile?
https://github.com/appc/spec/blob/master/aci/layout.go#L106